downloads/index.md

```diff
@@ -293,7 +293,7 @@ meta_descr: downloads.intro
                             <h3>{% t downloads.localsync %}</h3>
                             <p>{% t downloads.localsyncinfo %}</p>
                             <ul>
-                                <li><a class="ext-noicon" href="https://cakewallet.com" aria-label="GitHub icon" target="_blank"><img class="mob" style="width: 50px" src="/img/cakewallet.png" width="172" height="202" loading="lazy" alt="Cake Wallet Logo">Cake Wallet</a></li>
+                                <li><a class="ext-noicon" href="https://cakewallet.com" aria-label="GitHub icon" target="_blank"><img class="mob" style="width: 50px" src="/img/monerocom-wallet.png" width="172" height="202" loading="lazy" alt="Cake Wallet Logo">Cake Wallet</a></li>
```

Comment:
```suggestion
                                <li><a class="ext-noicon" href="https://cakewallet.com" aria-label="GitHub icon" target="_blank"><img class="mob" style="width: 50px" src="/img/monerocom-wallet.png" width="172" height="202" loading="lazy" alt="Monero.com Wallet Logo">Monero.com Wallet</a></li>
```

Parity with desktop

Reply:
Why the domain is cakewallet.com instead of monero.com tho?

Reply:
https://github.com/monero-project/monero-site/pull/2342#discussion_r1714347876

They both seem to be behind CF now, but CW com has fdroid link and both sites link to both wallets

Reply:
It makes more sense now to change it monero.com, it already look better and more concise. The only problem are the google trackers on the website, idk what is wrong the Cake people.

Reply:
Still dont have fdroid there

Reply:
sgp doesnt work for cake